src/components/block-editor/index.js

```diff
@@ -92,11 +93,13 @@ function BlockEditor({ settings: _settings }) {
 				</Sidebar.InspectorFill>
 				<div className="editor-styles-wrapper">
 					<BlockEditorKeyboardShortcuts.Register />
-					<WritingFlow>
-						<ObserveTyping>
-							<BlockList className="getdavesbe-block-editor__block-list" />
-						</ObserveTyping>
-					</WritingFlow>
+					<BlockTools>
```

Comment:
Can you point me to where in the Gutenberg project you found this pattern being used?

Reply:
I found it in the documentation of the block-editor handbook. I gave it a try and it was working.
Link: https://github.com/WordPress/gutenberg/blob/169dc1b3dab28db05ad7629b2a1243ed0f6953c9/packages/block-editor/README.md

Reply:
The component definition was also helpful.
https://github.com/WordPress/gutenberg/blob/169dc1b3dab28db05ad7629b2a1243ed0f6953c9/packages/block-editor/src/components/block-tools/index.js